### DISCUSSION TITLE
长江黄河，不会倒流 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#52] [General]
https://blog.1l1.icu/2023/11/03/chang-jiang-huang-he-bu-hui-dao-liu/ 

 Keep doing, keep loving❤.

**Comment by shiftz300** [comment]
In solitude, where we are least alone

**Comment by Muska-Ami** [comment]
> In solitude, where we are least alone

That's true
